packages/aserto/src/aserto/client/directory/__init__.py

```diff
@@ -0,0 +1,180 @@
+from typing import Tuple
+
+import grpc
+from aserto.directory.common.v2 import (
+    Object,
+    ObjectIdentifier,
+    ObjectTypeIdentifier,
+    PaginationRequest,
+    PermissionIdentifier,
+    Relation,
+    RelationIdentifier,
+    RelationTypeIdentifier,
+)
+from aserto.directory.exporter.v2 import ExporterStub
+from aserto.directory.importer.v2 import ImporterStub
+from aserto.directory.reader.v2 import (
+    CheckPermissionRequest,
+    CheckRelationRequest,
+    GetObjectRequest,
+    GetObjectsRequest,
+    GetObjectsResponse,
+    GetRelationRequest,
+    GetRelationsRequest,
+    GetRelationsResponse,
+    ReaderStub,
+)
+from aserto.directory.writer.v2 import (
+    DeleteObjectRequest,
+    DeleteRelationRequest,
+    SetObjectRequest,
+    SetRelationRequest,
+    WriterStub,
+)
+
+
+class Directory:
+    def __init__(self, *, address: str, api_key: str, tenant_id: str, ca_cert: str) -> None:
+        self._config = {
```

Comment:
I don't think you need this config object at all.
You can construct and store the metadata during `__init__` and if you change `_channel_credentials()` to take `cert` as an argument, you won't need to keep any of this config around.